xxxx-release-and-versioning.md

```diff
@@ -6,15 +6,25 @@ This document specifies a procedure for releasing and versioning to allow for st
 
 ## Overview
 
-The eWallet release is consist of a stable release and a nightly release. A stable release is meant to be run by a Provider that require stability and compatibility, while a nightly release is meant for development and testing propose. The stable releases should be versioned using [SemVer](https://semver.org/).
+The eWallet release is consist of a stable release and a development release. A stable release is meant to be run by a Provider that require stability and compatibility, while a development release is meant for development and testing propose. The stable releases should be versioned using [SemVer](https://semver.org/).
 
 ## Specifications
 
 ### Stable Release
 
-A stable release is a release that provides compatibility and stability guarantee and thus require an extensive end-to-end testing prior to release. Due to this, **we will only provide a stable release as a Docker image based on Alpine Linux**. The Docker image will be tagged with the release version and should use a `:latest` tag to refer to the latest stable release.
+A stable release is a release that provides compatibility and stability guarantee and must pass an extensive end-to-end testing and migration testing from supported versions prior to release.
 
-Only a the latest `MAJOR.MINOR.PATCH` version of the last 2 `MAJOR.MINOR` versions will be supported. However, we may provide security fixes to a non-supported versions as the team see fit. As an example, we shall support the following versions released in the past:
+| Test subject        | Required |
+| ------------------- | -------- |
+| Linting             | **Yes**  |
+| Unit testing        | **Yes**  |
+| Integration testing | **Yes**  |
+| End to end testing  | **Yes**  |
+| Migration testing   | **Yes**  |
+
+A stable release will be provided as a Docker image based on Alpine Linux. The Docker image will be tagged with the release version (e.g. `v1.0.3`) and will use a `:latest` tag to refer to the latest stable release.
+
+Only the latest `MAJOR.MINOR.PATCH` version of the last 2 `MAJOR.MINOR` versions will be supported. However, we may provide security fixes to non-supported versions as the team see fit. For demostration purpose, given that we released the following versions at a specific date, our support condition shall be:
```

Comment:
For demonstration purposes, given that we released the following versions at specific dates, our support condition shall be: